### PR TITLE
Issue #SB-23847 feat: Support for App Icon

### PR DESCRIPTION
**Pull request by rajnish-dargan**

### CODE REVIEW

File: projects/collection-editor-library/src/lib/services/editor/editor.service.ts [hunk 1]
```diff
@@ -260,5 +263,61 @@ export class EditorService {
     this.telemetryService.error(telemetryErrorData);
   }
 
+  getAssetMedia(req?: object) {
```
Comment (vaibhavbhuva) [inline projects/collection-editor-library/src/lib/services/editor/editor.service.ts:266]: @rajnish-dargan We have already created this `getAssetMedia` method in the `question.service.ts` file. 
Please check the same  and use it from here : `question.service.ts`

File: projects/collection-editor-library/src/lib/services/editor/editor.service.ts [hunk 2]
```diff
@@ -260,5 +263,61 @@ export class EditorService {
...
+          },
+          limit: 50,
+        }
+      }
+    };
+    reqParam.data.request = req ? _.merge({}, reqParam.data.request, req) : reqParam;
+    return this.publicDataService.post(reqParam);
+  }
+
+  createMediaAsset(req?: object) {
```
Comment (vaibhavbhuva) [inline projects/collection-editor-library/src/lib/services/editor/editor.service.ts:287]: @rajnish-dargan Check the above comment.

File: projects/collection-editor-library/src/lib/services/editor/editor.service.ts [hunk 3]
```diff
@@ -260,5 +263,61 @@ export class EditorService {
...
+
+  generatePreSignedUrl(req, contentId: any) {
+    const reqParam = {
+      url: `content/v3/upload/url/${contentId}`,
+      data: {
+        request: req
+      }
+    };
+    return this.publicDataService.post(reqParam);
+  }
```
Comment (vaibhavbhuva) [inline projects/collection-editor-library/src/lib/services/editor/editor.service.ts:321]: @rajnish-dargan We have already created this `getAssetMedia` method in the `question.service.ts` file. 
Please check the same  and use it from here : `question.service.ts`

cc: @pallakartheekreddy

Comment (rajnish-dargan) [reply projects/collection-editor-library/src/lib/services/editor/editor.service.ts]: @pallakartheekreddy Updated the changes please review once.

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**